praw/objects.py

```diff
@@ -652,7 +652,14 @@ def permalink(self):
 
     @property
     def replies(self):
-        """Return a list of the comment replies to this comment."""
+        """Return a list of the comment replies to this comment.
+        
```

Comment:
Looks like there's extra whitespace on this line. That appears to be why travis is failing.